libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/model/ArrowAddedValue.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.ui.maps.route.arrow.model
+
+import com.mapbox.geojson.FeatureCollection
+
+/**
+ * Represents the data for rendering the adding of route arrow(s)
+ *
+ * @param arrowShaftFeatureCollection a [FeatureCollection] for the arrow shafts source
+ * @param arrowHeadFeatureCollection a [FeatureCollection] for the arrow head source
+ */
+class ArrowAddedValue internal constructor(
```

Comment:
Same as above re: do we need all these 3 states? 🤔

Reply:
Not applicable after most recent refactor.